playbook/app/pb_kits/playbook/pb_form/form.rb

```diff
@@ -11,7 +11,7 @@ class Form < ::Playbook::KitBase
       prop :validate, type: Playbook::Props::Boolean, default: false
 
       def render_in(view_context, &block)
-        view_context.pb_form_with(form_options, &block)
+        view_context.pb_form_with(**form_options, &block)
```

Comment:
Just dropping a note here. This might be a place to check.

Reply:
Yeah this is one we had to actually change, because of the new splat operator stuff in Ruby 3.1. [Procs accepting a single rest argument and keyword arguments are no longer subject to autosplatting.](https://www.fastruby.io/blog/ruby/upgrades/upgrade-ruby-from-2.7-to-3.0.html) 
cc: @xjunior 

Yeah definitely agree, we should check the UI on forms and just make sure they are operating as they should.